doc/use.rst

```diff
@@ -47,13 +47,14 @@ Without validation
   from django.contrib.geos.geometries import GEOSGeometry
 
   layer = Layer.objects.create(name='Mushroom spot 2')
-  feature = Feature.objects.create(layer=layer,
-                                   geom=GEOSGeometry("POINT(0 0)")
+  feature = Feature(layer=layer,
+                    geom=GEOSGeometry("POINT(0 0)")
+  feature.clean()  # ok
```

Comment:
- Add information about modelform, viewsets that clean then save.
- Add information that you need to savec after the clean

Reply:
done